hvplot/converter.py

```diff
@@ -572,6 +578,8 @@ def __init__(
                 pass
         if cnorm is not None:
             plot_opts['cnorm'] = cnorm
+        if rescale_discrete_levels is not None:
+            plot_opts['rescale_discrete_levels'] = rescale_discrete_levels
```

Comment:
Need to guard this with a version check no?

Reply:
Yes indeed, I've added a version check for HoloViews>=1.15.0 in https://github.com/holoviz/hvplot/pull/758/commits/c5f325ac96cf45308ec411e50d95a885ac0c7e2f